<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loader</class>
 <widget class="qMRMLWidget" name="Loader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>406</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Text">
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="server">
     <property name="text">
      <string>https://clt-lab-t-7200.zhaw.ch:8000</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Access Token</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="token">
     <property name="text">
      <string>eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJqdXN0Z2l2ZW1ldGhlc3R1ZmYiLCJleHAiOjE3MTc4NDQwOTV9.2HX9Jkgnpz-b6oYoUQUoFiDODA4z3ubrmRPadOlgnRk</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_run_totalsegmentator">
     <property name="text">
      <string>Run TotalSegmentator</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Load File (hdf)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QLineEdit" name="load_file_uri">
          <property name="text">
           <string>Yonsei_3D/0000/Vol_ART_0000.hdf</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File URI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Run the algorithm.</string>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Infer</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLineEdit" name="infer_file_uri">
          <property name="text">
           <string>Yonsei_3D/0000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>File URI</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="infer_model_uri">
          <property name="text">
           <string>dualdomain_unet/8tjh2gzu</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Model URI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="inferButton">
        <property name="text">
         <string>Run &amp;&amp; Download</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
